operations/patch_util.go

```diff
@@ -421,6 +429,12 @@ func gitCommitMessages(base, ref string) (string, error) {
 	return gitCmd("log", args...)
 }
 
+// assumes base includes @{upstream}
+func gitLastCommitMessage(base, ref string) (string, error) {
+	args := []string{"--no-show-signature", "--pretty=format:%B", fmt.Sprintf("%s...%s", base, ref), "-n 1"}
```

Comment:
instead of `base...ref` I think we could use `HEAD`

Reply:
I think that logs everything. Don't we want `HEAD~1..HEAD`?

Reply:
Sounds good, I wasn't sure if there was a specific reason we did that elsewhere.

Reply:
it occurred to me we could do without the ref entirely, since `-n 1` will limit to the first message in the log.

Reply:
This has been removed.